src/OrchardCore.Modules/OrchardCore.Security/Extensions/PermissionsPolicyBuilderExtensions.cs

```diff
@@ -0,0 +1,37 @@
+using System;
+using Microsoft.Extensions.Options;
+using OrchardCore.Security;
+using OrchardCore.Security.Middlewares;
+
+namespace Microsoft.AspNetCore.Builder
+{
+    public static class PermissionsPolicyBuilderExtensions
+    {
+        public static IApplicationBuilder UsePermissionsPolicy(this IApplicationBuilder app)
```

Comment:
Shall we enable or diable all the features by default, while the intend the header should be present?

Reply:
I did few updates which I will push soon. We could disable the features by default, at least the header will be present all the time, otherwise it will be reported as missing header

Reply:
I did few updates which I will push soon. We could disable the features by default, at least the header will be present all the time, otherwise it will be reported as missing header

Reply:
I mentioned [here](https://github.com/OrchardCMS/OrchardCore/pull/11538#discussion_r851579740)